tests/jest/primary-navigation-layout/PrimaryNavigationLayout.test.jsx

```diff
@@ -0,0 +1,1044 @@
+import React from 'react';
```

Comment:
These tests replace the existing jest tests for the terra-application-navigation port. Coverage is spotty around logic that deals with notifications, as we do not expose notifications at this time, but otherwise solid.